src/datasets/features/DatasetVersionFileColumns/DatasetVersionFileColumns.tsx

```diff
@@ -0,0 +1,133 @@
+import { gql, useQuery } from "@apollo/client";
+import { useMemo } from "react";
+import Spinner from "core/components/Spinner";
+import { MetadataAttribute } from "graphql/types";
+import { camelCase } from "lodash";
+import Card from "core/components/Card";
+import DescriptionList from "core/components/DescriptionList";
+import { useTranslation } from "react-i18next";
+import Badge from "core/components/Badge";
+import { percentage } from "datasets/helpers/dataset";
+
+export type DatasetColumn = {
+  id: string;
+  columnName: string;
+  constantValues?: boolean;
+  dataType: string;
+  distinctValues: number;
+  key: string;
+  missingValues: number;
+  system: boolean;
+  uniqueValues?: number;
+  totalValues: number;
+};
+
+type DatasetVersionFileColumnsProps = {
+  file: any;
+};
+
+const DatasetVersionFileColumns = (props: DatasetVersionFileColumnsProps) => {
+  const { t } = useTranslation();
+  const { file } = props;
+  const { data, loading } = useQuery(
+    gql`
+      query DatasetVersionFileMetadata($id: ID!) {
+        datasetVersionFile(id: $id) {
+          id
+          attributes {
+            id
+            key
+            value
+            system
+          }
+        }
+      }
+    `,
+    {
+      variables: {
+        id: file.id,
+      },
+    },
+  );
+
+  const metadata: Array<DatasetColumn> = useMemo(() => {
+    if (!data?.datasetVersionFile.attributes) {
+      return [];
+    }
+
+    const { attributes } = data.datasetVersionFile;
+    return Object.values(
```

Comment:
This merge, for a column metadata object, all property into one, grouping by key and create a single object representing the column's complete set of properties.
E.g : 
```json
[
  {
    "id": "0e8a5901-3c2e-4b10-8d62-5acb1d87f0b1",
    "key": "25c49fb629ebd7438e7963850b1ce883.column_name",
    "value": "DATAFLOW",
    "system": true
  },
  {
    "id": "8cb8b6e4-8327-40c1-a073-7a066cc9538c",
    "key": "25c49fb629ebd7438e7963850b1ce883.data_type",
    "value": "string",
    "system": true
  }
]
````
Will be transformed into
```json
[
  {
    "id": "51438671-62f3-4295-8019-dcbecb8226e7",
    "key": "25c49fb629ebd7438e7963850b1ce883",
    "system": true,
    "columnName": "DATAFLOW",
    "dataType": "string"
  }
]
```

Reply:
Any reason why this is done on the Frontend and not the Backend instead ?

Reply:
Hum none that I know (it can be problematic for large dataset tho). Will report it to Nazar.

Reply:
Does it make sense to move it the backend then ? To avoid heavy computation on the client side and to be able to share the logic through our API layer 

I can understand we use this shortcut to deliver faster so I will not block the PR for that if that's the case and if we need to deliver it

Reply:
Yes, I agree ! I will create a ticket / Pr for that.